main/res/values-zh-rTW/strings.xml

```diff
@@ -993,7 +993,7 @@
     <string name="cache_terrain">地形</string>
     <string name="cache_rating">評分</string>
     <string name="cache_own_rating">你的評分</string>
-    <string name="cache_rating_of_new">%2$d 的 %1$d</string>
+    <string name="cache_rating_of_new">%2$d 的 %1$.1f</string>
```

Comment:
Changed it in the same way on Crowdin, so that this change wont get overwritten by Crowdin again.
Should fix the corresponding error.